docs/DESIGN.md

```diff
@@ -138,10 +137,10 @@ Right now, `@angular/tsc-wrapped` does not support the `"extends"` property of `
 Because of that, `ng-packagr` needs to support self-contained JSON configuration files for ngc.
 If auto-generating a tsconfig, `ng-packagr` would need to read its default values, merge that with the custom user tsconfig and copy the result to its working directory.
 
-The most important setting here is the `"files": []` property, which must contain exactly one file since `"flatModuleId"` and `"flatModuleOutFile"` options will also be used for flattended library indexes.
+The most important setting here is the `"files": []` property, which must contain exactly one file since `"flatModuleId"` and `"flatModuleOutFile"` options will also be used for flattened library indexes.
 The value for `"flatModuleId"` could be inferred by the library's name as given in `package.json`, `"flatModuleOutFile"` could be statically set to `"index"`.
 
-The `"lib"` property that denotes TypeScript language level support can be overriden through a `lib.languageLevel` property in `ng-package.json`.
+The `"lib"` property that denotes TypeScript language level support can be overridden through a `lib.languageLevel` property in `ng-package.json`.
```

Comment:
This can be removed since the option no longer exists.

Reply:
Done.